### PR TITLE
Fixed Bug: Links on the blog dont break to a new line on mobile - improved text flow

### PR DESCRIPTION
**Pull request by Shantanugupta43**
Fixed issue https://github.com/expressjs/expressjs.com/issues/1637

Links on the blog were not breaking on mobile view which was assigned as a bug.

I improved the alignment further to make sure the text gets a better flow.

<h2> Output of new change </h2>

image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1263,20 +1262,26 @@ h2 a {
...
+    flex-wrap: wrap;
     flex-direction: column;
-    align-items: center;
-    margin: 0;
+    align-items: flex-start;
+    margin-right: 0;
     padding-right: 10px;
   }
   #blog-doc .blog-details + p {
     display: flex;
```
Comment (bjohansebas) [inline css/style.css]: I think removing just this line would fix it, the other changes break the desktop design
```suggestion

```

**Comment by Shantanugupta43** [comment]
Still making the change @bjohansebas I think it still didnt fix when i previewed in mobile. Mobile preview on Desktop shows it's fixed but it's not when I tried on my actual mobile.

**Comment by bjohansebas** [comment]
Oh, okay, then put it in draft mode so we know that the issue is still being worked on.

**Comment by Shantanugupta43** [comment]
It should be fixed now @bjohansebas have a look and let me know.

File: css/style.css [hunk 2]
```diff
@@ -1263,20 +1263,27 @@ h2 a {
...
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+  
+
+
+  #blog-doc .blog-details + p a,
+  #blog-doc .blog-details + p a + span {
+    white-space: nowrap;
   }
```
Comment (bjohansebas) [inline css/style.css]:
```suggestion

```

before: 
image

after:
image

**Comment by Shantanugupta43** [comment]
Done 👍

**Review by bjohansebas — APPROVED** [review]
LGTM, thanks @Shantanugupta43

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**